packages/conf/conf.go

```diff
@@ -0,0 +1,224 @@
+package conf
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	toml "github.com/BurntSushi/toml"
+)
+
+const defaultConfigFile = "config.toml"
+
+// HostPort endpoint in form "str:int"
+type HostPort struct {
+	Host string // ipaddr, hostname, or "0.0.0.0"
+	Port int    // must be in range 1..65535
+}
+
+// Str converts HostPort pair to string format
+func (h HostPort) Str() string {
+	return fmt.Sprintf("%s:%d", h.Host, h.Port)
+}
+
+// DBConfig database connection parameters
+type DBConfig struct {
+	Type     string
+	Name     string
+	Host     string
+	Port     int
+	User     string
+	Password string
+}
+
+// StatsDConfig statd connection parameters
+type StatsDConfig struct {
+	Name     string // default "apla"
+	HostPort        // 127.0.0.1:8125
+}
+
+// Centrifugo connection params
+type CentrifugoConfig struct {
+	Secret string
+	URL    string
+}
+
+// SavedConfig parameters saved in "config.toml"
+type SavedConfig struct {
+	LogLevel    string // DEBUG, INFO, WARN, ERROR
+	LogFileName string // log file name relative to cwd or empty for stdout
+	InstallType string
+	NodeStateID string // default "*"
+	//
+	TestMode bool // ??? used in daemons/confirmations
+	//
+	StartDaemons string // comma separated list of daemons to start or empty for all or 'null'
+	//
+	KeyID       int64
+	EcosystemID int64
+	//
+	BadBlocks              string // ??? accessed once as json map
+	FirstLoadBlockchainURL string // ??? install -> blocks_colletcion
+	FirstLoadBlockchain    string // ??? install -> blocks_collection == 'file'
+	//
+	Daemon HostPort
+	HTTP   HostPort
+	DB     DBConfig
+	StatsD StatsDConfig
+	//
+	WorkDir        string // application work dir (cwd by default)
+	FirstBlockPath string // path to the first block file
+	PrivateDir     string // place for private keys files: NodePrivateKey, PrivateKey
+	//
+	Centrifugo CentrifugoConfig
+}
+
+// WebInstall web UI installation mode
+var WebInstall bool
```

Comment:
so, you deleted atomic bool installed, and make your own not-threadsafe bool WebInstall?